src/services/BiometricAuth.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @author Arthur Martello <arthur.martello@etu.utc.fr>
+ *
+ * @copyright Copyright (c) 2019, SiMDE-UTC
+ * @license GPL-3.0
+ */
+
+import React from 'react';
+import { Modal, Platform, Text, View } from 'react-native';
+import * as LocalAuthentication from 'expo-local-authentication';
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
+import * as Haptics from 'expo-haptics';
+import { BlurView } from 'expo-blur';
+import { _, BiometricAuth as t } from '../utils/i18n';
+import colors from '../styles/colors';
+import BlockTemplate from '../components/BlockTemplate';
+
+export const defaultSecurity = ['transfer', 'refill', 'badge-locking'];
+export const advancedSecurity = ['transfer', 'refill', 'badge-locking', 'app-opening'];
```

Comment:
Cool to use enumarations.
I would have suggest you to define each strings as a specific constant but it is not a problem.

Moreover, constants must be defined in snake capital case